drakrun/drakrun/test/test_lvm.py

```diff
@@ -0,0 +1,177 @@
+import pytest
+
+from drakrun.config import InstallInfo
+from drakrun.storage import LvmStorageBackend
+from drakrun.util import safe_delete
+import subprocess
+import os
+import logging
+import tempfile
+import secrets
+import string
+import parted
+import time
+
+
+@pytest.fixture(scope="module")
+def setup():
+    temp_file_name = tempfile.NamedTemporaryFile(delete=False).name
+    print(temp_file_name)
+    subprocess.run(['dd', 'if=/dev/zero', f'of={temp_file_name}', 'bs=1M', 'count=100'], stderr=subprocess.STDOUT, check=True)
+    loopback_file = subprocess.check_output(['losetup', '-f', '--show', temp_file_name], stderr=subprocess.STDOUT).decode().strip('\n')
+
+    # v is being added in start to ensure the lvm volume group starts with a letter
+    # don't know if it is required or not
+    volume_group = 'v' + ''.join(secrets.choice(string.ascii_letters + string.digits) for i in range(5))
```

Comment:
I'd prefer the tests to be deterministic. Please choose some simple name for the volume group.

```
# v is being added in start to ensure the lvm volume group starts with a letter
# don't know if it is required or not
```

I don't understand this. What's the origin of "v" ?

Reply:
No origin

> I'd prefer the tests to be deterministic. Please choose some simple name for the volume group.
> 
> ```
> # v is being added in start to ensure the lvm volume group starts with a letter
> # don't know if it is required or not
> ```
> 
> I don't understand this. What's the origin of "v" ?

Just a random letter (Inspired by "v"olume group), no specific origin as such. What name do you suggest for this then?